ui/src/components/ui/sonner.tsx

```diff
@@ -0,0 +1,38 @@
+/*
+ * The component is originally a part of the shadcn/ui library at https://github.com/shadcn-ui/ui.
+ * The library is licensed as set out below:
+ *
+ * Copyright (c) 2023 shadcn
+ *
+ * SPDX-License-Identifier: MIT
+ */
+
+import { useTheme } from 'next-themes';
+import { Toaster as Sonner } from 'sonner';
+
+type ToasterProps = React.ComponentProps<typeof Sonner>;
+
+const Toaster = ({ ...props }: ToasterProps) => {
+  const { theme = 'system' } = useTheme();
```

Comment:
Does this do anything without the `ThemeProvider` from `next-themes`?

Reply:
Probably not, at least for now, I've just taken the component as-it-is with the install command from shadcn-ui. I would think that once/if we take the light and dark modes into use, this will have an effect.